README.md

```diff
@@ -29,9 +29,6 @@ See [chaincode-docker-devmode](./chaincode-docker-devmode/README.rst)
 
 ## Documentation
 
-
-Note for internal use only: See the [technical specifications](https://github.com/SubstraFoundation/substra-spec/blob/master/technical_spec_substra.md#smartcontract).
-
```

Comment:
Maybe add a link to https://github.com/SubstraFoundation/substra-doc?

Reply:
`substra-doc` is now a legacy repo ;) The user doc is located in the `substra` repo.

Reply:
I forgot about that. So let's send him to substra then.

Reply:
@thibaultrobert What should the text for the link say?

Reply:
Well my first guess would be https://github.com/SubstraFoundation/substra but maybe @natct10 has a better suggestion as to where we can find Substra documentation.

Reply:
@thibaultrobert Sorry, I meant which text should we add a link to? like `[what do we put here???](some_link_to_susbtra)`

Reply:
Ok Mondays and I don't agree with each others.... `[Substra main repository]` ?
But at this point I would rather agree with the current state of the PR and remove that part completely. Maybe add a link to Substra but not as a `Note for internal use only`. So let me approve that for you and sorry for the noise

Reply:
Hi, a small piece of documentation related to smart contracts is on its way (but not ready yet!), what do you think about *temporarily* removing this internal link & redirecting towards the general doc and update this part later?